locales/en.json

```diff
@@ -15,6 +15,7 @@
     "search": "Search",
     "album": "Album",
     "podcast": "Podcast",
+    "unknown": "Unknown route",
```

Comment:
Should this say "Unknown page" instead? I don't know if "route" makes sense for end users

Reply:
I agree. Change it to "Page not found"